src/sentry/static/sentry/app/components/errors/notFound.jsx

```diff
@@ -4,7 +4,7 @@ import {t} from '../../locale';
 class NotFound extends React.Component {
   render() {
     return (
-      <div className="alert alert-block alert-error">
+      <div className="alert alert-block alert-error" style={{margin: '30px 0 10px'}}>
```

Comment:
is it possible to use scale here?

Reply:
@Chrissy That's a good thought, but margin would be a couple of pixels off due to `footer` not being on the same scale. I'll add a todo to remind us to update this when we do a broader scale pass on layout elements.